.github/workflows/merge-flow.yaml

```diff
@@ -0,0 +1,142 @@
+name: Common merge flow
+on:
+  workflow_call:
+    inputs:
+      go-version:
+        description: go version
+        required: true
+        type: string
+      upstream:
+        description: Upstream repo path in owner/repo format
+        required: true
+        type: string
+      downstream:
+        description: Downstream repo path in owner/repo format
+        required: true
+        type: string
+      downstream-branch:
+        description: Downstream branch to create PR
+        required: false
+        default: master
+        type: string
+      sandbox:
+        description: Sandbox repo path in owner/repo format. Used a base to create PR against downstream.
+        required: true
+        type: string
+      theirs:
+        description: Resets using upstream content during merge conflict.
+        required: false
+        default: ''
+        type: string
+      ours:
+        description: Resets using downstream content during merge conflict.
+        required: false
+        default: ''
+        type: string
+    secrets:
+      cloner-app-id:
+        description: Github ID of cloner app
+        required: true
+      cloner-app-private-key:
+        description: Github private key of cloner app
+        required: true
+      pr-app-id:
+        description: Github ID of PR creation app
+        required: true
+      pr-app-private-key:
+        description: Github private key of PR creation app
+        required: true
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    name: perform merge operation
+    steps:
+      - name: get latest upstream tag
+        id: upstream
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: ${{ inputs.upstream }}
+          excludes: prerelease, draft
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ inputs.downstream }}
+          fetch-depth: 0
+          ref: ${{ inputs.downstream-branch }}
+      - name: fetch upstream
```

Comment:
Is this just a debug step?

Reply:
Yes.

Reply:
nit: how  about naming it `show latest tag` ?